src/NzbDrone.Core/Notifications/Pushcut/PushcutProxy.cs

```diff
@@ -0,0 +1,64 @@
+using System;
+using System.Net.Http;
+using FluentValidation.Results;
+using NLog;
+using NzbDrone.Common.Http;
+using NzbDrone.Common.Serializer;
+
+namespace NzbDrone.Core.Notifications.Pushcut
+{
+    public interface IPushcutProxy
+    {
+        void SendNotification(string title, string message, PushcutSettings settings);
+        ValidationFailure Test(PushcutSettings settings);
+    }
+
+    public class PushcutProxy : IPushcutProxy
+    {
+        private readonly IHttpClient _httpClient;
+        private readonly Logger _logger;
+
+        public PushcutProxy(IHttpClient httpClient, Logger logger)
+        {
+            _httpClient = httpClient;
+            _logger = logger;
+        }
+
+        public void SendNotification(string title, string message, PushcutSettings settings)
+        {
+            const string URL = "https://api.pushcut.io/";
+            var request = new HttpRequestBuilder($"{URL}{settings.Secret}/notifications/{settings.NotificationName}")
+                .Accept(HttpAccept.Json)
+                .Build();
+            var payload = new PushcutPayload
```

Comment:
This would be the recommended way to do it with the request builder, with an API key instead of the global secret.

```suggestion
            var request = new HttpRequestBuilder($"https://api.pushcut.io/notifications/{notificationName}")
                .SetSegment("notificationName", settings.NotificationName)
                .Accept(HttpAccept.Json)
                .SetHeader("API-Key", Settings.ApiKey)
                .Build();

            var payload = new PushcutPayload
```